vispy/visuals/mesh.py

```diff
@@ -283,8 +283,8 @@ def _build_color_transform(self, colors):
         clim_func = 'float cmap(float val) { return (val - $cmin) / ($cmax - $cmin); }'
         if colors.ndim == 2 and colors.shape[1] == 1:
             fun = Function(clim_func)
-            fun['cmin'] = self.clim[0]
-            fun['cmax'] = self.clim[1]
+            fun['cmin'] = self._clim_values[0]
+            fun['cmax'] = self._clim_values[1]
```

Comment:
So this method/property was added and then never used? Wow, good find, but sad that it took this long to find it.

Reply:
Diving into the git history a bit I think it was used when added but lost during a refactor.